src/views/viewer/ChannelView/ChannelView.styles.ts

```diff
@@ -131,22 +131,33 @@ export const PaginationContainer = styled.div`
 `
 
 export const TabsContainer = styled.div`
-  display: grid;
   margin-bottom: ${sizes(8)};
-  gap: ${sizes(2)};
-  grid-template: 'tabs tabs tabs' 1fr 'search search search' auto 'sort sort sort' auto / 1fr 1fr;
-  align-items: baseline;
   ${media.xs} {
     padding-top: ${sizes(8)};
   }
   ${media.sm} {
     align-items: center;
-    border-bottom: solid 1px ${oldColors.gray[800]};
     gap: ${sizes(8)};
     grid-template: 1fr / auto 1fr 250px;
   }
 `
 
+export const UtilitiesContainer = styled.div`
+  display: grid;
+  gap: ${sizes(2)};
+  grid-template: 'search search search' auto 'sort sort sort' auto / 1fr 1fr;
```

Comment:
Sorry to bother you :D, but I'm pretty sure, that you don't need to set this template and grid-areas.

I opened PR https://github.com/wyrdrender/atlas/pull/3

Let me know what you think.

Reply:
Merged!